contracts/utils/TreasuryManager.sol

```diff
@@ -9,7 +9,7 @@ import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/I
 contract TreasuryManager is ITreasuryManager, Initializable, OwnableUpgradeable {
     address payable public treasury;
 
-    mapping(address token => uint256 fee) public fee;
+    uint256 public fee;
```

Comment:
In theory this should work (just need to switch off a check in openzeppelin's plugin), since the mapping's initial value will be replaced by a uint256. I should probably initialize it in a reinitializer function for a smoother upgrade flow

Reply:
Tested this locally, gonna mark this as resolved